bgpd/bgp_ecommunity.c

```diff
@@ -1026,3 +1026,133 @@ int ecommunity_fill_pbr_action(struct ecommunity_val *ecom_eval,
 		return -1;
 	return 0;
 }
+
+static unsigned int bgp_aggr_ecommunity_hash_key(void *data)
+{
+	struct ecommunity *ecommunity = data;
+
+	return jhash(ecommunity->val, ecommunity->size, 0xdcba4321);
+}
+
+static bool bgp_aggr_ecommunity_hash_cmp(const void *n1, const void *n2)
```

Comment:
isn't this ecommunity_cmp?

Reply:
I added a new function to safeguard the aggregation hash just in case the existing function goes through changes in the future.
I've made the changes to use the existing function.